.github/workflows/callbacks.yml

```diff
@@ -18,12 +18,13 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version: '1.23'
+          cache: false
```

Comment:
why?

Reply:
yes

Reply:
remnant of trying to fix linters I think. There was some warning and I added this so silence it, but I don't think the warning was much harm. I'll revert and see how it looks